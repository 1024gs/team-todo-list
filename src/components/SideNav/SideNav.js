import React from 'react';
import './SideNav.scss';

const SideNav = ({children}) => {
  return <div className="SideNav side-nav fixed">{children}</div>;
};

export default SideNav;
