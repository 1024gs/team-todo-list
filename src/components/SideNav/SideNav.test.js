import React from 'react';
import SideNav from './SideNav';
import {shallow} from 'enzyme';

const setup = (propOverrides) => {
  const props = Object.assign({children: <div className="unique" />}, propOverrides);
  const wrapper = shallow(<SideNav {...props} />);

  return {
    props,
    wrapper
  };
};

describe('SideNav', () => {
  it('renders', () => {
    const {wrapper} = setup();
    expect(wrapper).toMatchSnapshot();
  });

  it('renders children when passed in', () => {
    const {wrapper} = setup({children: <div className="avatar" />});
    expect(wrapper.contains(<div className="avatar" />)).toBe(true);
  });
});
