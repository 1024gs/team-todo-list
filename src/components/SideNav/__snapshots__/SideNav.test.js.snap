// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNav renders 1`] = `
<div
  className="SideNav side-nav fixed"
>
  <div
    className="unique"
  />
</div>
`;
